clients/onboarding/src/pages/v2company/OnboardingCompanyRegistration.tsx

```diff
@@ -45,6 +42,8 @@ import {
   validateRequired,
 } from "../../utils/validation";
 
+const countryItems = Lazy(() => allCountriesItems.filter(item => item.cca3 !== "USA"));
```

Comment:
can you just add a small comment explaining the filter?

Reply:
you're right will ask the product team

Reply:
I didn't have answers from product team and it seems there are in holidays. Waiting next week for an answer before merging this

Reply:
I just received an answer and add a comment here: 2c103dda19cb6b437fa890cac677fafce947c8fd